DQM/HcalCommon/interface/ElectronicsQuantity.h

```diff
@@ -311,6 +401,50 @@ namespace hcaldqm
 					return new FEDQuantity(vfeds);
 				}
 		};
+
+		// Crate quantity, initialized from emap (because it is not easy to turn a VME crate in an EID)
+		class CrateQuantity : public ElectronicsQuantity {
+			typedef std::map<int, uint32_t> CrateHashMap;
+		public:
+			CrateQuantity() {}
+			CrateQuantity(HcalElectronicsMap const * emap) : ElectronicsQuantity(fCrate, false) {
+				this->setup(emap);
+			}
+			CrateQuantity(std::vector<int> crates, CrateHashMap crateHashes) : ElectronicsQuantity(fCrate, false) {
+				this->setup(crates, crateHashes);
+			}
+			virtual ~CrateQuantity() {}
+
+			virtual void setup(HcalElectronicsMap const * emap);
+			virtual void setup(std::vector<int> crates, CrateHashMap crateHashes);
+			virtual int getValue(HcalElectronicsId const&);
+			virtual uint32_t getBin(HcalElectronicsId const&);
+
+			virtual int nbins() {
+				std::cout << "[debug] CrateQuantity: returning nbins = " << _crates.size() << std::endl;
```

Comment:
You probably did this for debugging purposes and forgot to disable afterwards.
Either way, cout should not be used.